.github/workflows/create-release.yml

```diff
@@ -39,3 +39,9 @@ jobs:
       run: node ./bin/create-github-release.js --tag ${{ steps.get_tag.outputs.latest_tag }} --repo-owner ${{ github.repository_owner }}
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    - run: npm run public-docs
```

Comment:
how did you test this or did you just trust the action will just work ™️

Reply:
Just using [act](https://github.com/nektos/act). Naturally this can only test publishing actions as dry runs, leading to a certain unfortunate distance between testing and production.